www/js/services/incidents_services.js

```diff
@@ -12,10 +12,10 @@ angular.module('mobility')
     area: "KAKAAKO",
     lat: 21.297355,
     lng: -157.861581,
-    icon: '../../img/ionic.png'
+    icon: 'img/ionic.png'
```

Comment:
Probably don't need this field anymore with your helper, right? It's not critical, since this is just mock data anyway.